Kitodo/src/main/webapp/WEB-INF/templates/includes/desktop/processesWidget.xhtml

```diff
@@ -22,7 +22,7 @@
                  styleClass="default-layout"
                  var="process">
         <p:column headerText="#{msgs.id}"
-                  width="50"
+                  width="70"
```

Comment:
I don't know if its possible to move the `width` declarations here and on the other place  to a style sheet property so the adjustment must be done only once and can maybe defined more granular.